pages/community/styleguide/code-examples.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Code examples
+weight: 20
+---
+
+# Code Examples
+
+## Guidelines
+
+Code examples which you add must be simple enough to understand and with good details to do something interesting and useful.
+
+You need to add practical code examples which adhere to industry standards as users will copy the sample code and run it in production. Be sure to include any warnings through comments which might occur during the code run.
+Make it clear if it is a snippet and not a complete example. Provide information about any packages or dependencies required to run the sample code.
+
+The code example should be self explanatory and easy to understand. The goal is to provide simplified working examples that can be understood as soon as possible, not necessarily to create efficient, brilliant code that impresses experts and has outstanding utility.
+
+
+## Formatting 
+
+### Code Length
+
+- Code lines shouldn't be so lengthy that reading them horizontally requires scrolling.
+
+- For the sake of readability, break large lines at appropriate moments, but never at the price of acceptable practises.
+
+- Add bullet points for better readability and understandability.
+
+
+### Colors
+
+- Using different colors for making the code more readable and understandable.
+Add different color to `tags` and `code` is a good practice.
+
+- Code blocks that show good habits to adopt are shown with a green check mark in the right corner, while those that show bad habits are shown with a white cross inside a red circle.
+
+- You different colors for bracket to know the starting and ending bracket of code block.
```

Comment:
```suggestion
### Colors
Color plays an important role in syntax highlighting, making code samples easier to read and understand code. Use color in a consistent and accessible way to make code examples accessible to everyone.

- Use a color scheme that is accessible: Ensure the color scheme meets accessibility standards. This means using colors that have enough contrast to be easily distinguishable from each other. 
- Use color to differentiate between elements: Color can be used to differentiate between different elements in your code, such as keywords, variables, and comments. This makes it easier to quickly identify and understand the different parts of the code.
- Test your code with color blindness simulators: There are many different types of color blindness, and it's important to ensure your code is readable for people with all types of color blindness. Use color blindness simulators to test how your code appears to users with different types of color blindness.

```

We do not currently have any styling in our docs' code snippets for green checkmarks showing right code block or red x's for wrong code blocks. We only use correct code snippets in our docs, so this example didn't apply. 

We also want to mention color accessibility, so I added that info.

Reply:
ohkkk Alejandra